pkg/features/features.go

```diff
@@ -47,9 +47,9 @@ const (
 
 	UseOSDisk Feature = "UseOSDisk"
 
-	// MountChangeDetection feature flag is used to enable/disable
-	// mount point and fs type change detection for block devices
-	MountChangeDetection = "MountChangeDetection"
+	// ChangeDetection is used to enable detecting changes to
+	// blockdevice size, filesystem, and mount-points.
+	ChangeDetection Feature = "ChangeDetection"
```

Comment:
This change also need to be made in the operator and chart files in the `deploy/` directory